sdk/appcenter/src/androidTest/java/com/microsoft/appcenter/utils/storage/DatabaseManagerAndroidTest.java

```diff
@@ -489,4 +490,40 @@ public void replaceByMultipleColumns() {
         }
         verify(listener).onCreate(any(SQLiteDatabase.class));
     }
+
+    @Test
+    public void testMultipleTables() {
+        final String firstTable = "firstTable";
+        final String secondTable = "secondTable";
+
+        /* Get instance to access database. */
+        DatabaseManager.Listener listener = mock(DatabaseManager.Listener.class);
+        DatabaseManager databaseManager = new DatabaseManager(sContext, "test-multiple-tables", firstTable, 1, mSchema, listener);
+        assertTrue(isTableExists(databaseManager.getDatabase(), firstTable));
+        assertFalse(isTableExists(databaseManager.getDatabase(), secondTable));
+
+        databaseManager.createTable(secondTable);
+        assertTrue(isTableExists(databaseManager.getDatabase(), firstTable));
+        assertTrue(isTableExists(databaseManager.getDatabase(), secondTable));
+
+        databaseManager.dropTable(firstTable);
+        assertFalse(isTableExists(databaseManager.getDatabase(), firstTable));
+        assertTrue(isTableExists(databaseManager.getDatabase(), secondTable));
+
+        databaseManager.dropTable(secondTable);
+        assertFalse(isTableExists(databaseManager.getDatabase(), firstTable));
+        assertFalse(isTableExists(databaseManager.getDatabase(), secondTable));
+    }
+
+    private boolean isTableExists(SQLiteDatabase db, String tableName) {
+        Cursor cursor = db.rawQuery("select DISTINCT tbl_name from sqlite_master where tbl_name = '" + tableName + "'", null);
```

Comment:
Would be better to just put a value and read it. With different schemas.

Reply:
Sure, I'll add write-read tests as well, but I still think it's valuable to check that there are indeed tables underneath.

Reply:
Still working on that new test. For some reason, the write to the second table doesn't actually add a record in there (though no exception and I checked in the DB browser, it's actually not there).

Reply:
Ok, that was a typo. Fixed it.

Reply:
If put/read works then the table has to exist. We can remove this code which is basically using unsecure query that can be picked up by a code scanner.